packages/migrate/loaders/loaders.ts

```diff
@@ -219,31 +221,31 @@ module.exports = function(j, ast) {
 	 * @returns {Node} ast - jscodeshift ast
 	 */
 
-	const loadersToRules = () =>
+	const loadersToRules = (_?: void): INode =>
 		ast
 			.find(j.Identifier)
 			.filter(checkForLoader)
-			.forEach(p => (p.value.name = "rules"));
+			.forEach((p: INode) => (p.value.name = "rules"));
 
 	/**
 	 * Convert `loader` and `loaders` to Array of {Rule.Use}
 	 *
 	 * @returns {Node} ast - jscodeshift ast
 	 */
 
-	const loadersToArrayExpression = () =>
+	const loadersToArrayExpression = (_?: void): INode | void =>
 		ast
 			.find(j.ObjectExpression)
-			.filter(path => utils.findObjWithOneOfKeys(path, ["loader", "loaders"]))
+			.filter((path: INode) => utils.findObjWithOneOfKeys(path, ["loader", "loaders"]))
 			.filter(
-				path =>
+				(path: INode) =>
 					utils.safeTraverse(path, [
 						"parent",
 						"parent",
 						"node",
 						"key",
-						"name"
-					]) === "rules"
+						"name",
+					]) === "rules",
 			)
 			.forEach(createArrayExpressionFromArray);
```

Comment:
would be nice to have type check on this if possible

Reply:
Where exactly? We're already doing type checking for `createArrayExpressionFromArray` function expression.